pkgs/by-name/ca/caido/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchurl
+, appimageTools
+, makeWrapper
+}:
+
+let
+  pname = "caido";
+  version = "0.29.2";
+  name = "caido-desktop-linux-v${version}-e0f8102b";
+  nameExecutable = pname;
```

Comment:
Is this really necessary?

Reply:
`name` or `nameExecutable`?

Reply:
nameExecutable

Reply:
Does it now look [good](https://github.com/NixOS/nixpkgs/pull/270664/commits/333185fdb8fb471d11b0f0ad71b35477a07399b2)?

Reply:
It looks better. You still need to squash the commits.

Reply:
[Done](https://github.com/NixOS/nixpkgs/pull/270664/commits/7c6b4185db3d7afecae0aba7d0d409c336e84178)